tests/test_paperqa.py

```diff
@@ -457,6 +458,22 @@ def test_llm_read_json(example: str):
     assert llm_read_json(example) == {"example": "json"}
 
 
+def test_llm_read_json_newlines():
```

Comment:
Can you add a docstring or comment that newlines are an edge case that broke us

Reply:
Thanks! Done